.github/workflows/terraform-plan-apply-manual.yml

```diff
@@ -32,10 +31,10 @@ jobs:
       - name: Check changed files
         id: calc_files_diff
         run: |
+          echo "env var: ${{ github.event.inputs.env }}"
           git fetch origin main:localmain
 
-          # Gets changed json files excluding the json schema
-          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema")
+          export FILES=$(git diff --name-only localmain | grep .json | grep "\/${{ github.event.inputs.env }}_.")
```

Comment:
so why is this exception needed?

Reply:
This checks the following..
Added code that when an env para is passed via GH the code looks for a definition file that matches the env param passed with a _ as a delimiter.

It looks like Ive erroneously removed the | grep -v "trial-json-schema" filter..maybe I missed that when merging? Ill re-add.

Reply:
actually on second thoughts - the grep -v "trial-json-schema" filter isnt needed as grep "\/${{ github.event.inputs.env }}_." filter json files with a [env]_.json format. Therefore no additional commit necessary